docs/sections/pipeline_samples/tutorials/GenerateSentencePair.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>We could add a !!! note at the end of the section just to indicate the components as we had mentioned some time ago: <a href="https://haystack.deepset.ai/tutorials/27_first_rag_pipeline" target="_blank">https://haystack.deepset.ai/tutorials/27_first_rag_pipeline</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>